lib/services/ios-provision-service.ts

```diff
@@ -1,11 +1,19 @@
 import * as mobileprovision from "ios-mobileprovision-finder";
 import { createTable } from "../common/helpers";
 
-const months = ["Jan", "Feb", "Marc", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 function formatDate(date: Date): string {
 	return `${date.getDay()} ${months[date.getMonth()]} ${date.getFullYear()}`;
 }
 
+/**
+ * Formats the argument so it can easily be copied from the terminal and provided as value for an option.
+ * @param arg The string to format.
+ */
+function cmdEscape(arg: string): string {
```

Comment:
Maybe [quoteString](https://github.com/telerik/mobile-cli-lib/blob/master/helpers.ts#L99-L105) can do the same work for you?